content/sips/sip-161.md

```diff
@@ -48,72 +50,155 @@ On the topic of scaling and decentralizing the Synthetix Protocol, the onboardin
+Depending on the performance of the lead and squad, the CCC may use its discretion to allocate additional bonuses for their work.
 
-The lead will be responsible for organizing a timeline, plan and a "squad" of Core Contributors who will be working on this SIP.
+In terms of prioritization, the Core Contributors will also be able to signal on a platform such as canny.io on which SIPs they think are most important, this will help foster a collaborative relationship between the Core Contributors and the CCC.
 
-The CCC will then have the discretion to appoint further bonuses to the "lead" and the "squad" depending on their performance and efficiency.
+_Non-Technical Work_
 
-In terms of prioritisation, the Core Contributors will also be able to signal on a platform such as canny.io on which SIPs they think are most important, this will help foster a collaborative relationship between the Core Contributors and CCC members.
+Whilst SIPs are predominately for specifying technical protocol work, the non-technical work still remains an important aspect of running a successful protocol. The CCC will be responsible for scoping non-technical work in a similar fashion that is done with technical SIPs, this may require that there be non-technical leaders voted in for the Core Contributor Committee role.
+
+_Protocol Maintenance Work_
+
+The advantages to having Core Contributors who are engaged by the Protocol is that it provides the Protocol with access to resources quickly. Apart from well-defined projects in SIPs, the CCC will also need to schedule resources for ad-hoc tasks like technical upgrades, bug-fixes and other maintenance task in which the SIP process does not account for.
 
 #### Protocol Road-map and Vision
 
-For the long-term sustainability of the Synthetix Protocol, the Core Contributor Committee will be responsible for collecting and discussing feedback and ideas for the management of the long-term road-map and vision of the Synthetix Protocol. This means interacting with the different DAOs, community members and also ecosystem partners to help synthesize the future of the Synthetix Protocol.
+For the long-term sustainability of the Synthetix Protocol, the Core Contributor Committee will be responsible for collecting and discussing ideas for the long-term vision of the Synthetix Protocol. This means that the CCC will be responsible for interfacing with all the Synthetix Governing Bodies and Stakeholders to help synthesize a direction in which the Protocol should aim to follow.
+
+#### Core Contributor Committee Stipends & Transparency
+
+The Core Contributor Committee will be a incentivized role funded by the Treasury Council, the CCC stipends amounts will be openly shared. However, the amounts decided on by the Committee in regards to new and existing Core Contributors will remain accessible by only those involved internally.
```

Comment:
May want to make this more clear. The way I interpret it is that the stipends will be shared openly amongst all CC's but not with the Community. Will Spartan Council members also get access to this info?

Reply:
no only the CCC members know abou the stipends assigned to the CC, but the CCC stipends are all public